tests/IndexerTest.php

```diff
@@ -0,0 +1,196 @@
+<?php
+declare(strict_types=1);
+
+namespace Tests\LoyaltyCorp\Search;
+
+use LoyaltyCorp\Search\Exceptions\AliasNotFoundException;
+use LoyaltyCorp\Search\Indexer;
+use LoyaltyCorp\Search\Interfaces\ClientInterface;
+use LoyaltyCorp\Search\Interfaces\Helpers\EntityManagerHelperInterface;
+use LoyaltyCorp\Search\Interfaces\ManagerInterface;
+use Tests\LoyaltyCorp\Search\Stubs\ClientStub;
+use Tests\LoyaltyCorp\Search\Stubs\Handlers\HandlerStub;
+use Tests\LoyaltyCorp\Search\Stubs\Helpers\EntityManagerHelperStub;
+use Tests\LoyaltyCorp\Search\Stubs\ManagerStub;
+
+/**
+ * @covers \LoyaltyCorp\Search\Indexer
+ */
+class IndexerTest extends TestCase
+{
+    /**
+     * Ensure the search handler index + '_new' index gets created
+     *
+     * @return void
+     *
+     * @throws \EoneoPay\Utils\Exceptions\InvalidDateTimeStringException
+     */
+    public function testAliasGetsCreated(): void
+    {
+        $elasticClient = new ClientStub();
+        $indexer = $this->createInstance($elasticClient);
+
+        $indexer->create(new HandlerStub());
+
+        self::assertSame(['valid_new'], $elasticClient->getCreatedAliases());
+    }
+
+    /**
+     * Ensure the cleaning process only disregards indices unrelated to search handlers
+     *
+     * @return void
+     */
+    public function testCleaningIndicesDoesNotRemoveUnrelatedIndices(): void
+    {
+        // #1. Have random indices exist (and aliases?)
+        // #2. Have search handler indices exi
+        $client = new ClientStub(
+            null,
+            null,
+            [['name' => 'unrelated-index'], ['name' => 'irrelevant-index'], ['name' => 'valid-123']]
+        );
+        $indexer = $this->createInstance($client);
+
+        $indexer->clean([new HandlerStub()]);
+
+        // unrelated-index and irrelevant-index should not be touched, because they are unrelated to search handlers
+        self::assertSame(['valid-123'], $client->getDeletedIndices());
```

Comment:
expected is part of arrange

Reply:
The comment is explaining why the expected array only contains `'valid-123'`, and not including `'irrelevant-index', 'unrelated-index'`

Reply:
Yes, but we define expected variables as part of *A*AA, not AA*A* :)